internal/ent/hooks/trustcentersetting.go

```diff
@@ -0,0 +1,147 @@
+package hooks
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"entgo.io/ent"
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+	"github.com/theopenlane/core/internal/ent/generated"
+	"github.com/theopenlane/core/internal/ent/generated/hook"
+	"github.com/theopenlane/core/pkg/objects"
+	"github.com/theopenlane/iam/fgax"
+)
+
+var ErrTooManyLogoFiles = errors.New("too many logo files uploaded, only one is allowed")
+
+func HookTrustCenterSetting() ent.Hook {
+	return hook.On(func(next ent.Mutator) ent.Mutator {
+		return hook.TrustCenterSettingFunc(func(ctx context.Context, m *generated.TrustCenterSettingMutation) (generated.Value, error) {
+			zerolog.Ctx(ctx).Info().Msg("trust center setting hook")
```

Comment:
Either use `Debug` so its there for local dev, ore remove

```suggestion
			zerolog.Ctx(ctx).Debug().Msg("trust center setting hook")
```